20190618/custom_model.sql

```diff
@@ -0,0 +1,5 @@
+SELECT * FROM creditcardfraud
```

Comment:
For figures like `column_all.png`, can just refer to the file using relative path: `../20190618/xx.png`